src/hooks/setup.ts

```diff
@@ -8,4 +8,236 @@ export default function setup() {
 	registerSystemSettings();
 
 	game.nimble.conditions.initialize();
+
+	if (foundry.applications?.sheets) {
```

Comment:
Is there a reason why we're making sheets we don't touch resizable? Mostly because it adds more overhead and maintenance when foundry changes things.

Reply:
Additionally, I modified the code so that it only changes the DEFAULT_OPTIONS to include resizable property.